addons/l10n_fr_pos_cert/static/src/js/Popups/ClosePosPopup.js

```diff
@@ -0,0 +1,39 @@
+odoo.define('l10n_fr_pos_cert.ClosePosPopup', function (require) {
+    'use strict';
+
+    const ClosePosPopup = require('point_of_sale.ClosePosPopup');
+    const Registries = require('point_of_sale.Registries');
+
+    const PosFrCertClosePopup = (ClosePosPopup) =>
+        class extends ClosePosPopup {
+            getLimitDate() {
```

Comment:
instead of "getLimitDate", do a function "sessionIsOutdated" (or something like this) that already compare the limit date with dat.now()

Reply:
Done.